tests/data/test_case_080/.surface.bin.yml

```diff
@@ -62,7 +62,7 @@ fmu:
         anything: something
 file:
   relative_path: realization-33/iter-0/share/results/maps/volantis_gp_base--amplitude.gri
-  absolute_path: /some/absolute/path//realization-33/iter-0/share/results/maps/volantis_gp_base--amplitude.gri
+  absolute_path: /Users/RAYW/devel/sumo-repos/tmp/fmu-sumo-uploader/tests/data/test_case_080/.surface.bin.yml
```

Comment:
Replace with something more generic?

Reply:
Should not be part of the commit... running the tests cause the test data to be modified.